src/lib/Group.ts

```diff
@@ -109,6 +122,10 @@ export class Group<
     )
   }
 
+  /**
+   * Executes a network request to fetch the latest messages associated with the group
+   * and save them to the local state.
+   */
```

Comment:
Maybe mention here as well that it also fetches the latest membership changes (since they are technically a message)

Reply:
good call, added here - 20a69ad8fb6cf8365f35f6b93545d50a2ad545a6